src/OmegaPoker.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0
+// The RegaPoker
+//
+// Copyright (C) 2020 Maker Ecosystem Growth Holdings, INC.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.6.12;
+
+interface IlkReg {
+    function list() external returns (bytes32[] memory);
+    function pip(bytes32) external returns (address);
+}
+
+interface Chainlog {
+    function getAddress(bytes32) external returns (address);
+}
+
+interface OSMLike {
+    function src() external returns (address);
+}
+
+contract OmegaPoker {
+
+    bytes4   constant ssel = 0x1504460f;  // "poke(bytes32)"
+    bytes4   constant osel = 0x18178358;  // "poke()"
+    bytes4   constant rsel = 0x2e7dc6af;  // "src()"
```

Comment:
I'm pretty unpicky about var names, but these are maybe a little too opaque for even my tastes :joy:

Reply:
varnames embiggened